TLM/.editorconfig

```diff
@@ -97,7 +96,7 @@ indent_size=4
 indent_style=space
 indent_size=4
 
-[*.{appxmanifest,asax,ascx,aspx,build,config,cshtml,csproj,dbml,discomap,dtd,htm,html,jsproj,lsproj,master,njsproj,nuspec,proj,props,razor,resw,resx,skin,StyleCop,targets,tasks,vb,vbproj,xaml,xamlx,xml,xoml,xsd}]
+[*.{appxmanifest,asax,ascx,aspx,build,config,cshtml,dbml,discomap,dtd,htm,html,jsproj,lsproj,master,njsproj,nuspec,proj,props,razor,resw,resx,skin,StyleCop,targets,tasks,vb,vbproj,xaml,xamlx,xml,xoml,xsd}]
 indent_style=space
 indent_size=4
-tab_width=4
```

Comment:
This tab with was 4 I changed it to 8. @aubergine10 said that tabwidth = 4 is dangerous.